pkg/activator/handler/handler.go

```diff
@@ -196,7 +195,6 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		configurationName := revision.Labels[serving.ConfigurationLabelKey]
 		serviceName := revision.Labels[serving.ServiceLabelKey]
 		a.reporter.ReportRequestCount(namespace, serviceName, configurationName, name, httpStatus, attempts, 1.0)
```

Comment:
Is it possible to move request count reporting to the new metric handle as well? Could we pass `attempts`, as well as `namespace`, `serviceName`, etc to out layer handle in a similar way like how `httpStatus` is passed?

Reply:
well I think that was the original intent.
But I think reporting metric where it is captured is actually fine, even if we report in more than one place.